docs/pages/server-access/guides/openssh.mdx

```diff
@@ -10,7 +10,11 @@ accept SSH certificates dynamically issued by a Teleport CA.
 
 Using Teleport and OpenSSH has the advantage of getting you up
 and running, but in the long run, we would recommend replacing `sshd` with `teleport`.
-We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://goteleport.com/blog/openssh-vs-teleport/)
+`teleport` SSH servers have support for multiple features that are incompatible with OpenSSH:
+
+- [Dynamically updating node labels from the output of configured commands](../../management/admin/labels.mdx)
```

Comment:
Labels are Teleport specific, so I'm not sure why this would be meaningful to an OpenSSH user. Is the expectation here that OpenSSH users would understand what Teleport labels are?

Reply:
My thoughts were having an explicit list of what is and isn't supported with agentless nodes would be useful to set user's expectations, what do you think, is that not necessary?